### PR TITLE
Feature add ping command and time command

### PR DESCRIPTION
**Pull request by Anonymous390**
- Added Ping and Time command

### CODE REVIEW

**Review by abir0 — APPROVED** [review]

File: Pipfile [hunk 1]
```diff
@@ -6,6 +6,7 @@ name = "pypi"
 [packages]
 prompt-toolkit = "~=3.0"
 pygments = "~=2.9"
+pythonping = "*"
```
Comment (DavinderJolly) [inline Pipfile:9]: change * to ~=current major and minor version for example 2.9.0 would become ~=2.9

File: main/commands.py [hunk 2]
```diff
@@ -209,3 +213,23 @@ def move_file(self, src_path: str, dest_path: str) -> None:
                     print("Invalid option please enter the correct option")
         else:
             src_path_obj.replace(dest_path_obj)
+
+    def ping_addr(self, addr: str) -> None:
+        """Spawn a subprocess to ping the address
+
+        Args:
+            addr (str): The address of the website or the IP
```
Comment (DavinderJolly) [inline main/commands.py:221]: probably remove types from docstrings to stay consistent with all the other functions as we are already type hinting functions

File: main/commands.py [hunk 3]
```diff
@@ -209,3 +213,23 @@ def move_file(self, src_path: str, dest_path: str) -> None:
...
+            no_of_packets (Optional[int], optional): No of packets to send. Defaults to 4.
+        """
+        ping(f"{addr}", verbose=True)
+
+    def get_time(self, format: Optional[str] = None) -> None:
+        """Show the time
+
+        Args:
```
Comment (DavinderJolly) [inline main/commands.py:229]: same here probably remove type hinting (you can keep just optional if you wish though)

File: main/main.py [hunk 4]
```diff
@@ -106,28 +106,44 @@ def call_commands(self, input_text: str) -> None:
                 print("Usage: DELTREE dir_name")
 
         elif command == "move":
-            if len(command_input) >= 2:
+            if len(command_input) == 2:
                 self.commands.move_file(command_input[0], command_input[1])
             else:
                 print("Usage: MOVE source_path destination_path")
```
Comment (DavinderJolly) [inline main/main.py:112]: make the print a bit more easy to understand  instead of `source path` `destination path` if possible

Comment (DavinderJolly) [reply main/main.py:112]: just something like current path and new path

**Review by DavinderJolly — CHANGES_REQUESTED** [review]

File: main/commands.py (hunk 2 above)
Comment (Anonymous390) [reply main/commands.py:221]: but yeah its the generator that creates that and I cba to change it lol

File: main/main.py (hunk 4 above)
Comment (Anonymous390) [reply main/main.py:112]: uhhhh, duh thats literally how its on every single function and that space may not be nice for the user to understand

**Review by DavinderJolly — APPROVED** [review]

**Pull request merged**